tests/testsuite/cargo_command.rs

```diff
@@ -252,69 +271,68 @@ fn find_closest_alias() {
     fs::write(
         &my_home.join("config"),
```

Comment:
Would you mind updating to `config.toml` so that we can avoid the "[ROOT]/my_home/config is deprecated in favor of config.toml" warning.

```suggestion
        &my_home.join("config.toml"),
```

Reply:
done